distgen/main.go

```diff
@@ -90,6 +90,11 @@ func main() {
 		globDistJson = os.Args[3]
 	}
 
+	file, ok := os.LookupEnv("DIST_FILE")
+	if ok != false {
```

Comment:
I rather simplify this if statement as:
```suggestion
	if ok {
```